.github/workflows/gate.yml

```diff
@@ -71,23 +71,85 @@ jobs:
           echo "ANSYS_DPF_ACCEPT_LA=Y" >> $GITHUB_ENV
           echo "ANSYSLMD_LICENSE_FILE=1055@${{ secrets.LICENSE_SERVER }}" >> $GITHUB_ENV
 
-      - name: Setup Python
+      - name: "Setup Python"
         uses: actions/setup-python@v4.6.0
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: "Build Package"
-        uses: ansys/pydpf-actions/build_package@v2.3
+      - name: "Install requirements"
+        run: pip install -r requirements/requirements_build.txt
+
```

Comment:
what is the purpose of the gate.yml workflow now ?

Reply:
> what is the purpose of the gate.yml workflow now ?

@cbellot000  ah yes I'll remove it